pyproject.toml

```diff
@@ -69,11 +69,13 @@ sphinx-copybutton = { version = "^0.5.0" }
 sphinx-gallery = { version = "^0.11.0" }
 sphinx_rtd_theme = { version = "^1.0.0" }
 graphviz = { version = "^0.20.1" }
-dowhy = { version = "^0.8" }
 ipython = { version = "^7.4.0" }
 nbsphinx = { version = "^0.8" }
-scikit-learn = { version = "^1.1.0" }
-pandas = { version = "^1.1" }
+dowhy = { version = "^0.8" }
+scikit-learn = { version = "^1.1.0" }  # needed in dowhy's package
+pandas = { version = "^1.1" }  # needed in dowhy's package
+joblib = { version = "^1.1.0" }  # needed in dowhy's package
+tqdm = { version = "^4.64.0" }  # needed in dowhy's package
```

Comment:
This should get automatically pulled in through dowhy. Unless something is wrong (and you're not directly using these package in pywhy-graphs), this should not be necessary.

Reply:
Trying this now...